src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -63,6 +66,9 @@
     public static final String ASSIGN_DATE_TIME = " " + PREFIX_DATE + VALID_DATE + " " + PREFIX_TIME + VALID_TIME;
     public static final String ASSIGN_TIME = " " + PREFIX_TIME + VALID_TIME;
     public static final String ASSIGN_DATE = " " + PREFIX_DATE + VALID_DATE;
+    public static final String REMARK_DESC_AMY = " 1 " + PREFIX_REMARK + VALID_REMARK_AMY;
+    public static final String REMARK_DESC_BOB = " 2 " + PREFIX_REMARK + VALID_REMARK_BOB;
+    public static final String REMARK_DESC_EMPTY = " 1 " + PREFIX_REMARK + " ";
```

Comment:
Do you think it is better if we include the index directly in the tests instead of here?
Because currently it seems a bit inconsistent with the rest. 

Not really a big issue tho. Let me know what you think

Reply:
I would think this helps abstract information.
The only problem is that we need to arrange it well.
Maybe we can clean it up in some time.

Reply:
Yeah I agree with Wanlin on the abstraction part but I can also see the point about consistency. Hmm, I'll change it to match the style used in the rest of the files instead 👍🏻